wpilibc/athena/src/CameraServer.cpp

```diff
@@ -175,16 +175,17 @@ void CameraServer::Serve() {
   address.sin_addr.s_addr = htonl(INADDR_ANY);
   address.sin_port = htons(kPort);
 
-  if (bind(sock, (struct sockaddr*)&address, sizeof(address)) == -1)
+  if (bind(sock, reinterpret_cast<struct sockaddr*>(&address),
```

Comment:
It's a bit odd to say `reinterpret_cast<struct sockaddr*>` as the struct is redundant in C++ (it's needed in C).  I'd recommend removing the `struct` part here and in line 187.